touchforms/formplayer/static/formplayer/script/xforgasm.js

```diff
@@ -119,6 +163,9 @@ function WebFormSession(params) {
     self.serverRequest = function (params, callback, blocking) {
         var that = this;
         var url = that.urls.xform;
+        if (params.action === 'submit-all' && this.NUM_PENDING_REQUESTS) {
+            self.taskQueue.addTask(params.action, self.serverRequest, arguments, self)
```

Comment:
definitely confusing and will be refactored at a later time. for now i'll switch `this.NUM_PENDING_REQUESTS` to `self.` for consistency